src/GLideNHQ/TxQuantize.h

```diff
@@ -32,6 +32,48 @@ class TxQuantize
 private:
   int _numcore;
 
+  const volatile unsigned char One2Eight[2] =
+  {
+      0, // 0 = 00000000
+     255, // 1 = 11111111
+  };
+
+  const volatile unsigned char Five2Eight[32] =
+  {
+    0, // 00000 = 00000000
+    8, // 00001 = 00001000
+    16, // 00010 = 00010000
+    25, // 00011 = 00011001
+    33, // 00100 = 00100001
+    41, // 00101 = 00101001
+    49, // 00110 = 00110001
+    58, // 00111 = 00111010
+    66, // 01000 = 01000010
+    74, // 01001 = 01001010
+    82, // 01010 = 01010010
+    90, // 01011 = 01011010
+    99, // 01100 = 01100011
+    107, // 01101 = 01101011
+    115, // 01110 = 01110011
+    123, // 01111 = 01111011
+    132, // 10000 = 10000100
+    140, // 10001 = 10001100
+    148, // 10010 = 10010100
+    156, // 10011 = 10011100
+    165, // 10100 = 10100101
+    173, // 10101 = 10101101
+    181, // 10110 = 10110101
+    189, // 10111 = 10111101
+    197, // 11000 = 11000101
+    206, // 11001 = 11001110
+    214, // 11010 = 11010110
+    222, // 11011 = 11011110
+    230, // 11100 = 11100110
+    239, // 11101 = 11101111
+    247, // 11110 = 11110111
+    255  // 11111 = 11111111
+  };
+
```

Comment:
Why add them to header?

Would not it better to make them as static variables in TxQuantize.cpp ?